internal/infrastructure/event/consumer.go

```diff
@@ -56,7 +56,7 @@ func NewConsumer(config *config.Kafka) (*kafka.Consumer, error) {
 	if err = consumer.SubscribeTopics(config.Topics, nil); err != nil {
 		return nil, err
 	}
-	log.Info().Msgf("Consumer subscribed to topics: %s", strings.Join(config.Topics, ","))
+	slog.Info("Consumer subscribed to topics", strings.Join(config.Topics, ","))
```

Comment:
`!BADKEY`

Reply:
Fixed.

Reply:
the current state update the initial conversation; resolving this; and addressing a change a didn't realize in a quick ticket; instead of `"strings"` use `"topics"` for the tag of the log message.